recipe/meta.yaml

```diff
@@ -17,22 +18,23 @@ build:
 
 requirements:
   host:
+    - python >=3.8
+    - hatchling
     - pip
-    - python >=3.7,<3.9
   run:
+    - python >=3.8
     - attrs
     - boto3
     - cachetools
     - httpx
-    - importlib_resources >=1.1.0
-    - morecantile >=3.1,<4.0
     - numexpr
     - numpy
+    - morecantile >=3.1,<4.0
     - pydantic
     - pystac >=0.5.4
-    - python >=3.7,<3.9
-    - rasterio >=1.1.7
-    - rio-color
+    - rasterio >=1.3.0
+    # - color-operations
```

Comment:
```suggestion
    - pip:
      - color-operations
```

Reply:
conda-forge does not using pip in the recipe. We better get `color-operations` on conda-forge.

Reply:
> We better get color-operations on conda-forge.

up to you 👍